modules/Auth/Services/AuthService.php

```
@@ -109,10 +110,10 @@ public function validateOtp(ValidateOtpDTO $validateOtpDTO)
     public function ResetPassword(ResetPasswordCommand $resetPasswordCommand)
     {
         $token = $this->verficationDataRepository->findOneBy(["token" => $resetPasswordCommand->getToken()]);
```

The condition `true ||` was removed, which is correct as it was bypassing the security check. However, consider adding a proper error response when the token is invalid or doesn't have permission to reset password, rather than silently failing.